### PR TITLE
Move ArHelper to TopologicalInventory::Persister

### PR DESCRIPTION
**Pull request by agrare**
Move the ArHelper module under TopologicalInventory::Persister and move
into a method so that it isn't accidentally run when required.

### CODE REVIEW

File: lib/topological_inventory/persister/ar_helper.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+module TopologicalInventory
+  module Persister
+    module ArHelper
+      def self.load_environment!
+        require "active_record"
+
+        app_root = Pathname.new(__dir__).join("../../../")
+        $LOAD_PATH << File.join(app_root, "app", "models")
```
Comment (bdunne) [inline lib/topological_inventory/persister/ar_helper.rb]: `app_root.join("app", "models")`?

File: lib/topological_inventory/persister/ar_helper.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+module TopologicalInventory
+  module Persister
+    module ArHelper
+      def self.load_environment!
+        require "active_record"
+
+        app_root = Pathname.new(__dir__).join("../../../")
```
Comment (bdunne) [inline lib/topological_inventory/persister/ar_helper.rb]: `Pathname.new(__dir__).join("../../../").expand_path`

File: lib/topological_inventory/persister/ar_helper.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        $LOAD_PATH << File.join(app_root, "app", "models")
+        $LOAD_PATH << File.join(app_root, "app", "models", "concerns")
+
+        require "application_record"
+        Dir[File.expand_path("app/models/**/*.rb", app_root)].each { |f| require f }
+
+        app_env = ENV["RAILS_ENV"] || "development"
+
+        require "yaml"
+        connection_spec = ENV["DATABASE_URL"] || YAML.load_file(File.expand_path("config/database.yml", app_root))[app_env]
```
Comment (bdunne) [inline lib/topological_inventory/persister/ar_helper.rb]: `app_root.join("config/database.yml")`

**Comment by agrare** [comment]
@bdunne updated

File: lib/topological_inventory/persister/ar_helper.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    module ArHelper
+      def self.load_environment!
+        require "active_record"
+
+        app_root = Pathname.new(__dir__).join("../../../").expand_path
+        $LOAD_PATH << app_root.join("app", "models")
+        $LOAD_PATH << app_root.join("app", "models", "concerns")
+
+        require "application_record"
+        Dir[File.expand_path("app/models/**/*.rb", app_root)].each { |f| require f }
```
Comment (bdunne) [inline lib/topological_inventory/persister/ar_helper.rb]: Don't need the `expand_path` here either `app_root.join("app/models/**/*.rb")`

Comment (agrare) [reply lib/topological_inventory/persister/ar_helper.rb]: Done

**Pull request merged**